### DISCUSSION TITLE
Wildfly Integration

### DISCUSSION THREAD

**Discussion by amoscatelli** [jberet/jberet-jpa-repository#13] [Ideas]
Hi @chengfang 
I would like to make a pull request with my Wildfly integration.

Anyway I am seeing that Wildfly 27 beta is still using legacy jberet, still using javax.batch.

Do you know when it's going to be migrated to jakarta ?
Thank you in advance

**Comment by chengfang** [comment]
@amoscatelli It should happen soon. WildFly 27 is nearing the end of development and will not be ready for new features like that. I think your PR will need to target WildFly 28 at the earliest.

**Comment by amoscatelli** [comment]
Ok thank you !
I'll wait for wildfly main codebase to be migrated to jakarta.batch, then I'll update my integration and then I'll issue the pull request !

**Comment by chengfang** [comment]
Once https://github.com/wildfly/wildfly/pull/16041 is merged, WildFly will be on jakarta.batch API.

**Comment by amoscatelli** [comment]
@chengfang 
Here we are !
https://github.com/wildfly/wildfly/pull/16085
